pkg/apis/core/validation/events.go

```diff
@@ -18,12 +18,17 @@ package validation
 
 import (
 	"fmt"
+	"reflect"
 	"time"
 
+	"k8s.io/api/core/v1"
+	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/kubernetes/pkg/apis/core"
+	eventsv1beta1 "k8s.io/kubernetes/pkg/apis/events/v1beta1"
```

Comment:
Since all we need is the SchemeGroupVersion, referencing the `k8s.io/api/...` package here and in validation_test.go would keep you from needing to open up more import boss exceptions. The following passes import restriction checks:

```diff
diff --git a/cmd/cloud-controller-manager/.import-restrictions b/cmd/cloud-controller-manager/.import-restrictions
index 64b3f15fd31..962ddf3bd50 100644
--- a/cmd/cloud-controller-manager/.import-restrictions
+++ b/cmd/cloud-controller-manager/.import-restrictions
@@ -21,8 +21,6 @@ rules:
       - k8s.io/kubernetes/pkg/apis/core/v1
       - k8s.io/kubernetes/pkg/apis/core/v1/helper
       - k8s.io/kubernetes/pkg/apis/core/validation
-      - k8s.io/kubernetes/pkg/apis/events/v1
-      - k8s.io/kubernetes/pkg/apis/events/v1beta1
       - k8s.io/kubernetes/pkg/apis/scheduling
       - k8s.io/kubernetes/pkg/capabilities
       - k8s.io/kubernetes/pkg/client/leaderelectionconfig
diff --git a/pkg/apis/core/validation/events.go b/pkg/apis/core/validation/events.go
index 4b1a6e4d371..680d0e21f9c 100644
--- a/pkg/apis/core/validation/events.go
+++ b/pkg/apis/core/validation/events.go
@@ -28,7 +28,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/kubernetes/pkg/apis/core"
-	eventsv1beta1 "k8s.io/kubernetes/pkg/apis/events/v1beta1"
+	eventsv1beta1 "k8s.io/api/events/v1beta1"
 )
 
 const (
diff --git a/pkg/apis/core/validation/events_test.go b/pkg/apis/core/validation/events_test.go
index 54309bd91a4..7a2b513e32e 100644
--- a/pkg/apis/core/validation/events_test.go
+++ b/pkg/apis/core/validation/events_test.go
@@ -23,8 +23,8 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/apis/core"
-	eventsv1 "k8s.io/kubernetes/pkg/apis/events/v1"
-	eventsv1beta1 "k8s.io/kubernetes/pkg/apis/events/v1beta1"
+	eventsv1 "k8s.io/api/events/v1"
+	eventsv1beta1 "k8s.io/api/events/v1beta1"
 )
 
 func TestValidateEventForCoreV1Events(t *testing.T) {
diff --git a/test/e2e/framework/.import-restrictions b/test/e2e/framework/.import-restrictions
index 0e45afced7e..9b0fc023db9 100644
--- a/test/e2e/framework/.import-restrictions
+++ b/test/e2e/framework/.import-restrictions
@@ -20,8 +20,6 @@ rules:
       - k8s.io/kubernetes/pkg/apis/core/v1/helper
       - k8s.io/kubernetes/pkg/apis/core/v1/helper/qos
       - k8s.io/kubernetes/pkg/apis/core/validation
-      - k8s.io/kubernetes/pkg/apis/events/v1
-      - k8s.io/kubernetes/pkg/apis/events/v1beta1
       - k8s.io/kubernetes/pkg/apis/extensions
       - k8s.io/kubernetes/pkg/apis/networking
       - k8s.io/kubernetes/pkg/apis/policy
```

Reply:
Got it, thanks!